### PR TITLE
Fix a Relu(Conv) test by making sure Relu(Conv) won't be fused

### PR DESCRIPTION
**Pull request by Krovatkin**

### CODE REVIEW

**Comment by Krovatkin** [comment]
TESTNOW

File: test/cpu_fusion.cpp [hunk 1]
```diff
@@ -1003,46 +1003,90 @@ TEST(cpu_fusion, fuse_conv_relu)
...
+    return result_vectors;
+}
+
+TEST(cpu_fusion, conv_relu_n2c1h2w2_2)
+{
+    Shape shape_a{2, 1, 6, 6};
+    Shape shape_weights{1, 1, 2, 2};
 
-    cf->call({_conv_relu, _relu}, {_a, _weights});
-    EXPECT_TRUE(test::all_close(read_vector<float>(_conv_relu), read_vector<float>(_relu)));
```
Comment (pruthviIntel) [inline test/cpu_fusion.cpp]: may be better to use full name here, make_interpreter_function. ? :)

File: test/cpu_fusion.cpp [hunk 2]
```diff
@@ -1003,46 +1003,90 @@ TEST(cpu_fusion, fuse_conv_relu)
...
+    auto make_int_function = [shape_a, shape_weights]() {
```
Comment (pruthviIntel) [inline test/cpu_fusion.cpp]: may be better to use full name here, make_interpreter_function. ? :)

Comment (Krovatkin) [reply test/cpu_fusion.cpp]: @pruthviIntel , actually, it looks so aligned everywhere else 😄 
```cpp
    auto int_results = execute(int_f, args, "INTERPRETER");
    auto cpu_results = execute(cpu_f, args, "CPU");
```

how strongly do you feel about this?

Comment (pruthviIntel) [reply test/cpu_fusion.cpp]: hmm, true!. its okay, then.

**Pull request merged**